cirq/aqt/aqt_sampler.py

```diff
@@ -187,8 +187,7 @@ def run_sweep(self,
             resolver.
         """
         meas_name = 'm'  # TODO: Get measurement name from circuit. Issue #2195
-        circuit = (program.to_circuit()
-                   if isinstance(program, schedules.Schedule) else program)
+        circuit = program
```

Comment:
guess we don't want to change interface.  Could just use program throughout instead of circuit?

Reply:
Done.